frontend/src/app/shared/components/dashboard/dashboard.component.html

```diff
@@ -1,6 +1,11 @@
 <bf-navigation></bf-navigation>
-<bf-profile></bf-profile>
+<div class="container-fluid">
+  <div class="row my-5">
+    <bf-profile class="col-12 col-md-6"></bf-profile>
+    <bf-tweet-card class="col-12 col-md-6"></bf-tweet-card>
```

Comment:
Use "col" (defaulted to 12) instead of "col-12".

- [ ] Change "col-12" -> "col".